streams/quickstart/pom.xml

```diff
@@ -22,7 +22,7 @@
     <groupId>org.apache.kafka</groupId>
     <artifactId>streams-quickstart</artifactId>
     <packaging>pom</packaging>
-    <version>2.3.2-SNAPSHOT</version>
+    <version>2.3.3-SNAPSHOT</version>
```

Comment:
Same as above.

Reply:
Same as above, it was already on 2.3.2, so I undid this change.